indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/PartialSegmentMergeTask.java

```diff
@@ -266,6 +271,7 @@ private Set<DataSegment> mergeAndPushSegments(
             persistDir,
             0
         );
+        long mergeFinishTime = System.nanoTime();
```

Comment:
It might be nice to add a message here stating that the merge completed.  It can be nice to see it in its own line when you are watching logs for progress.

Reply:
Added a message upon completion of merge stage + fixed the message to add all interpolated values as per convention + removed objectMapper usage in the log.